packages/pglite/src/base.ts

```diff
@@ -78,6 +89,27 @@ export abstract class BasePGlite
 
   // # Concrete implementations:
 
+  async _initArrayTypes() {
+    if (this.#arrayTypesInitialized) return
+    this.#arrayTypesInitialized = true
+
+    const types = await this.query<{ oid: number; typarray: number }>(`
```

Comment:
I'm not quite sure what this does, assuming it sets up array serialization/parsing but can't quite understand why we can't do that statically? Could you leave a comment pointing to PG documentation or briefly describing why we do this?

Reply:
I've copied this from Postgres.js. I'm not 100% sure that all the values you get back are static between Postgres versions.

will add a comment.